docs/en/deploy/Running-Alluxio-On-Docker.md

```diff
@@ -264,13 +267,11 @@ $ docker exec ${container_id} cat /opt/alluxio/conf/alluxio-env.sh
 ### Run in High-Availability Mode
 
 A lone Alluxio master is a single point of failure. To guard against this, a production
-cluster should run multiple Alluxio masters and use internal leader election or Zookeeper-based leader election.
-One of the masters will be elected leader and serve client requests.
-If it dies, one of the remaining masters will become leader and pick up where the previous master left off.
+cluster should run multiple Alluxio masters in [High Availability mode]({{ '/en/deploy/Running-Alluxio-On-a-HA-Cluster.html' | relativize_url }}).
 
-#### Internal leader election
+#### Option A: Embedded Journal
 
-Alluxio uses internal leader election by default.
+Alluxio uses embedded journal by default.
```

Comment:
This part should point to HA doc as much as possible

Reply:
When this doc was written it was under our guidelines to refer to `Embedded Journal` as `Internal Leader Election` as often as possible. Is there a reason to change that here?

Reply:
Not at all. I'm not aware of that guideline...